IBPSA/Resources/ReferenceResults/Dymola/IBPSA_Fluid_Movers_Examples_ClosedLoop_y.txt

```diff
@@ -13,5 +13,5 @@ statistics-simulation=
-conPID.u_m=[0e+00, 5.224461555480957e-01, 6.58552348613739e-01, 7.132970690727234e-01, 7.352567911148071e-01, 7.441821098327637e-01, 7.477734684944153e-01, 7.490530610084534e-01, 7.496210336685181e-01, 7.498462796211243e-01, 7.499350309371948e-01, 7.499726414680481e-01, 7.499886155128479e-01, 4.751942157745361e-01, 3.413117229938506e-01, 2.861240208148956e-01, 2.642614543437958e-01, 2.554828524589539e-01, 2.520939111709595e-01, 2.507979869842529e-01, 2.50303715467453e-01, 2.501152157783508e-01, 2.500442564487457e-01, 2.500174045562744e-01, 2.500132024288177e-01, 2.499998956918716e-01, 6.068893074989319e-01, 6.925624012947084e-01, 7.269492745399476e-01, 7.407504320144653e-01, 7.462765574455261e-01, 7.484378218650818e-01, 7.490848302841187e-01, 7.496749758720398e-01, 7.499066591262817e-01, 7.499626278877258e-01, 7.499846220016479e-01, 7.499938607215881e-01, 4.751964211463928e-01, 3.413126468658447e-01, 2.861241400241852e-01, 2.642627656459808e-01, 2.554904222488403e-01, 2.52095103263855e-01, 2.507992386817932e-01, 2.503031492233276e-01, 2.501151263713837e-01, 2.500432431697845e-01, 2.500159740447998e-01, 2.50006377696991e-01, 2.500022053718567e-01, 6.072960495948792e-01, 6.925585865974426e-01, 7.269400954246521e-01, 7.407394647598267e-01, 7.460858225822449e-01, 7.485131621360779e-01, 7.494041323661804e-01, 7.497590184211731e-01, 7.499040365219116e-01, 7.499611377716064e-01, 7.499846816062927e-01, 7.499934434890747e-01, 4.751962721347809e-01, 3.413125574588776e-01, 2.861245572566986e-01, 2.642573714256287e-01, 2.55480021238327e-01, 2.520941197872162e-01, 2.507894933223724e-01, 2.503139674663544e-01, 2.500575184822083e-01, 2.499309778213501e-01, 2.500254511833191e-01, 2.49990701675415e-01, 2.500195503234863e-01, 6.036717295646667e-01, 6.925596594810486e-01, 7.269438505172729e-01, 7.407376170158386e-01, 7.462510466575623e-01, 7.484816312789917e-01, 7.493463158607483e-01, 7.497832179069519e-01, 7.499030828475952e-01, 7.499610781669617e-01, 7.499841451644897e-01, 7.499934434890747e-01, 4.751960933208466e-01, 3.413124978542328e-01, 2.861242592334747e-01, 2.642591595649719e-01, 2.554605901241302e-01, 2.520957291126251e-01, 2.507982850074768e-01, 2.503035962581635e-01, 2.501151859760284e-01, 2.500427663326263e-01, 2.500166594982147e-01, 2.50006228685379e-01, 2.500018179416656e-01]
```

Comment:
I assume you checked all plots of the changes reference results outside of the Electrical package. Did Dymola change something in the solver or settings, or why do the results fail?

On this note, would it be easy to change the buildingspy script to have a one-value-per-line format for the reference results? This would make reviewing these in git easier.

Reply:
The changes are small and due to a change in numerics between the dymola versions.

The advantage of one series by line is that it shows easily which trajectory changes, and files don't get thousand's of lines long.